src/components/Button.js

```diff
@@ -108,33 +117,43 @@ class BaseButton extends Component {
   static defaultProps = {
     buttonType: BUTTON_TYPES.ACTION,
     fallBackClasses: '',
+    labelSize: LABEL_SIZE.MEDIUM,
     testID: 'test-id',
     width: null,
   };
 
+  /**
+   * Get the backbutton
+   *
+   * @return {element|null}
+   */
+  backButton() {
+    const { classes, buttonType } = this.props;
+
+    return buttonType === BUTTON_TYPES.BACK ? (
+        <img src={ backPng } srcSet={ backSvg } className={ classes.backImage }/>
+      )
+      : null;
+  }
+
   render() {
-    const { classes, dataTest, fallBackClasses, label, buttonType, onClick, testID } = this.props;
-    let backButton = null;
+    const { classes, dataTest, fallBackClasses, label, labelSize, buttonType, onClick, testID } = this.props;
 
-    if (buttonType === BUTTON_TYPES.BACK) {
-      backButton = <img src={ backPng } srcSet={ backSvg } className={ classes.backImage }/>;
-    }
+    // ` ${fallBackClasses}` is there to make it backwards compatible,
+    // it doesn't not have an other function
+    const rootClass = classes.root + ' ' + classes[ `root${ buttonType }` ] + ` ${ fallBackClasses }`;
+    const labelClass = classes.label + ' ' + classes[ `label${ buttonType }` ] + ' ' + classes[ `${ labelSize }Label` ];
```

Comment:
I think we should use template literals instead of concat-ing string. I would do either:
```
const rootClass = `${classes.root} ${classes[`root${buttonType}`]} ${fallBackClasses}`;
const labelClass = `${classes.label} ${classes[`label${buttonType}`]} ${classes[`${labelSize}Label`]}`;
```
or
```
const rootButtonTypeClass = classes[`root${buttonType}`];
const labelButtonTypeClass = classes[`label${buttonType}`];
const labelSizeClass = classes[`${labelSize}Label`];
const rootClass = `${classes.root} ${rootButtonTypeClass} ${fallBackClasses}`;
const labelClass = `${classes.label} ${labelButtonTypeClass} ${labelSizeClass}`;
```
(and ignore [my comment](https://github.com/saucelabs/sample-app-web/pull/16#discussion_r309925810) in https://github.com/saucelabs/sample-app-web/pull/16).

Reply:
Will check this one, thanks!